migrations/1753376254.sh

```diff
@@ -0,0 +1,21 @@
+echo "Installing omarchy-monitor-settings for Hyprland monitor control"
+
+if ! command -v omarchy-monitor-settings &>/dev/null; then
+  # Install Go if not already installed
+  if ! command -v go &>/dev/null; then
+    mise use -g go@lts
```

Comment:
Here you use Go through `mise`, but on the installation script you install Go through `yay`. I think it might be better to pick `mise` or `yay` go install Go and stick with it.